joint_limits:
  left_hip_joint:
    has_position_limits: true
    min_position: -0.2
    max_position: 1.0
    has_velocity_limits: true
    max_velocity: 0.2

    has_soft_limits: true
    soft_lower_limit: 0.1
    soft_upper_limit: 0.9
    k_position: 10.0
    k_velocity: 10.0